pkgs/top-level/all-packages.nix

```
@@ -28281,6 +28281,10 @@ in
 
   elmerfem = callPackage ../applications/science/physics/elmerfem {};
 
+  professor = callPackage ../applications/science/physics/professor {
+    python = python3;
+  };
+
```

```suggestion
  professor = callPackage ../applications/science/physics/professor { };

```
I think we could replace python with python3 but not that important.

---

Right, I was going for python2 compatibility before I knew that py3-only iminuit was a dependency.